examples/Bunch/cmakeFlags

```diff
@@ -35,6 +35,16 @@ flags[2]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_INCLUDE_FIELDBACKGROUND
 flags[3]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_SINGLE_PARTICLE=true"
 flags[4]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DENABLE_SYNCHROTRON_PHOTONS=1"
 flags[5]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_FILTER_GAMMA=1"
+flags[6]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_RADWINDOW=radWindowFunctionTriangle"
```

Comment:
be aware that compile time tests should cover syntax compatibility for now, we are not yet at RT tests which could scan the influence of windows and form factors regularly and check they don't change unexpectedly